deepchem/utils/test/data/gnina_log.txt

```diff
@@ -0,0 +1,25 @@
+              _             
```

Comment:
As a quick question for @dkoes, can we assume that this log file format is stable? The PR assumes that the log file will follow this set structure to read/write log files gnina generates.

Reply:
I can't guarantee it in perpetuity, but there are no changes planned that would invalidate the processing.  If you want a more robust solution, extract the scores from the output sdf.